doc/api/dgram.md

```diff
@@ -1,5 +1,7 @@
 # UDP / Datagram Sockets
 
+<!--introduced_in=v0.10.0-->
```

Comment:
I'll fix this, it might not technically matter but it should be the correct version